controller/work_item_link_type_blackbox_test.go

```diff
@@ -190,10 +196,11 @@ func (s *workItemLinkTypeSuite) TestCreateAndDeleteWorkItemLinkType() {
 	_ = test.DeleteWorkItemLinkTypeOK(s.T(), s.svc.Context, s.svc, s.linkTypeCtrl, *workItemLinkType.Data.Relationships.Space.Data.ID, *workItemLinkType.Data.ID)
 }
 
-//func (s *workItemLinkTypeSuite) TestCreateWorkItemLinkTypeBadRequest() {
-//	createPayload := s.createDemoLinkType("") // empty name causes bad request
-//	_, _ = test.CreateWorkItemLinkTypeBadRequest(s.T(), nil, nil, s.linkTypeCtrl, createPayload)
-//}
+// func (s *workItemLinkTypeSuite) TestCreateWorkItemLinkTypeBadRequest() {
```

Comment:
Actually I did some refactoring of some names and accidentally modified already commented out code. I will make this code more useful by running the `Validate` function on certain structs to see that the design works. When I wrote this I thought I could test it with an actual request but it gets blocked by goa even before doing so. Hence, the validate.

Fixed in  d0a2334d588a7b2bcf8db1b02dec02a239324dd7